backend/services/client.js

```diff
@@ -62,8 +62,9 @@ exports.signupService = async function({ email, password, type, name, lastName }
     throw new AppError(Messages.RETURN_MESSAGES.ERR_CLIENT_IS_ALREADY_REGISTERED);
   }
 
+  const encryptedPassword = sha256(password + "t2KB14o1");
   const newClient = (
-    await ClientDataAccess.createClientDB({email, password, type, name, lastName})
+    await ClientDataAccess.createClientDB({email, encryptedPassword, type, name, lastName})
```

Comment:
This will create a document named `encryptedPassword`. To fix that:
```suggestion
    await ClientDataAccess.createClientDB({email, password: encryptedPassword, type, name, lastName})
```